<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WindowUtama</class>
 <widget class="QMainWindow" name="WindowUtama">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>666</width>
    <height>379</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>#</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>24</number>
    </property>
    <property name="topMargin">
     <number>24</number>
    </property>
    <property name="rightMargin">
     <number>24</number>
    </property>
    <property name="bottomMargin">
     <number>6</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>8</number>
      </property>
      <item>
       <widget class="QPushButton" name="btnAmbilStok">
        <property name="text">
         <string>Ambil stok barang</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnTambahStok">
        <property name="text">
         <string>Tambah stok barang</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>12</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QTableView" name="tableBarang"/>
    </item>
    <item>
     <spacer name="verticalSpacer_4">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>12</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>8</number>
      </property>
      <item>
       <widget class="QPushButton" name="btnHapusBarang">
        <property name="text">
         <string>Hapus jenis barang</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnPerbaruiBarang">
        <property name="text">
         <string>Perbarui jenis barang</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnTambahBarang">
        <property name="text">
         <string>Tambah jenis barang</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>666</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actRefresh"/>
    <addaction name="actLogout"/>
    <addaction name="separator"/>
    <addaction name="actTentang"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actRefresh">
   <property name="text">
    <string>Refresh data</string>
   </property>
  </action>
  <action name="actLogout">
   <property name="text">
    <string>Logout</string>
   </property>
  </action>
  <action name="actTentang">
   <property name="text">
    <string>Tentang</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
